### ISSUE TITLE
What is going on here? Extremely off-balance values.

### ISSUE THREAD

**Issue by whyyoulookinatmaname** [taraniselsu/TacLifeSupport#45]
Excerpt from OxygenLarge: 
// --- resource parameters ---
    RESOURCE
    {
        name = Oxygen
        amount = 486496.4
        maxAmount = 486496.4
    }
}

Excerpt from LifeSupport:
// --- resource parameters ---
    RESOURCE
    {
        name = Food
        amount = 113.2
        maxAmount = 113.2
    }
    RESOURCE
    {
        name = Water
        amount = 74.8
        maxAmount = 74.8
    }
    RESOURCE
    {
        name = Oxygen
        amount = 11466.9
        maxAmount = 11466.9
    }

In short, what is going on here. Multiple redownloads; problem is persistent. Total reinstall of KSP, problem persists. Problem even persists in a fresh download folder. How has nobody else noticed this?
